app/src/main/java/kr/ac/kaist/team888/locator/Locator.java

```diff
@@ -680,6 +681,24 @@ public void manipulateSkeleton(double flatteningControl, double ariseControl) {
     }
   }
 
+  public void applySlant(double slantControl) {
+    double slantRadian = slantControl * Math.PI / 2 * SLANT_MAX;
+    for (ArrayList<BezierCurve> contour : contours) {
+      for (int i = 0; i < contour.size(); i++) {
+        BezierCurve bezierCurve = contour.get(i);
+        Vector2D[] points = bezierCurve.getPoints();
+        for (int j = 0; j < points.length; j++) {
+          Vector2D point = points[j];
+
+          double x = point.getX()
+              + (point.getY() - locatorRegion.getMinY()) / (Math.tan(Math.PI / 2 - slantRadian));
+
+          bezierCurve.setPoint(j, new Vector2D(x, point.getY()));
```

Comment:
`setPoint()` in for loop is not efficient because each `setPoint()` call re-calculates the polynomials of the curve. How about makes `setPoints()` in `BezierCurve` and calls it?